lib/commands/link.js

```diff
@@ -108,6 +108,7 @@ class Link extends ArboristWorkspaceCmd {
     // npm link should not save=true by default unless you're
     // using any of --save-dev or other types
     const save =
+      !Boolean(this.npm.config.argv[3] === '--no-save') &&
```

Comment:
This could be anywhere on argv

Reply:
This needs to be a proper check against the `save` config value, both for if it has been changed from the default AND if it's been changed to true.

Reply:
Fixed